### PR TITLE
Replace sameCluster by targetCluster in create new event command

### PR DESCRIPTION
**Pull request by alexeykazakov**
Part of https://issues.redhat.com/browse/SANDBOX-724

### CODE REVIEW

**Review by mfrancisc — APPROVED** [review]
Looks good 👍 

I have only one question - can the clusters in the ksctl.yaml be out of synch with the registered clusters ?

File: pkg/cmd/create_social_event.go [hunk 1]
```diff
@@ -101,13 +112,13 @@ func CreateSocialEvent(ctx *clicontext.CommandContext, startDate, endDate, descr
...
-			SpaceTier:         spaceTier,
-			Description:       description,
-			PreferSameCluster: preferSameCluster,
+			StartTime:     metav1.NewTime(start),
+			EndTime:       metav1.NewTime(end),
+			MaxAttendees:  maxAttendees,
+			UserTier:      userTier,
+			SpaceTier:     spaceTier,
+			Description:   description,
+			TargetCluster: targetCluster,
```
Comment (MatousJobanek) [inline pkg/cmd/create_social_event.go:121]: this is a name used in ksctl.yaml file - it's a short-cut that we use in command-line, but it doesn't have any meaning in the cluster. 

We need to use the name of the ToolchainCluster CR instead. You can use either the current approach that is used in the `retarget` command https://github.com/kubesaw/ksctl/blob/2cf7e16c94f0227f58cba32d33f3e4fa98cc0fe2/pkg/cmd/retarget.go#L104-L106 or get the ToolchainCluster CR and get the name from it.

Comment (alexeykazakov) [reply pkg/cmd/create_social_event.go]: Good catch. Addressed here: https://github.com/kubesaw/ksctl/pull/59/commits/89b7a0fbd944ec862e33646db5b12739dbae30dc

**Comment by alexeykazakov** [comment]
> I have only one question - can the clusters in the ksctl.yaml be out of synch with the registered clusters ?

Hm... what do you mean? :)

**Comment by mfrancisc** [comment]
> Hm... what do you mean? :)

I meant checking that there is actually a toolchaincluster resource for the cluster where the event needs to happen, if the ksctl.yaml file  might be out of date and have a cluster that doesn't exist or that is not configured yet for provisioning or any other reason for which the cluster might not be ready for provisioning users.

**Comment by MatousJobanek** [comment]
@mfrancisc the logic should ideally list ToolchainCluster CRs and find the correct one based on the api URL from ksctl.yaml file. This is something that should be fixed in other commands as well, so let's address it in a separate PR.

**Review by MatousJobanek — APPROVED** [review]
cool :+1:

**Pull request merged**